cpp/include/cudf/stream_compaction.hpp

```diff
@@ -285,9 +280,9 @@ std::unique_ptr<table> unordered_drop_duplicates(
  *
  * @return number of distinct consecutive groups in the column
```

Comment:
```c++
* @return number of distinct consecutive groups in the column
```
It might be best to avoid the word `distinct` here, to avoid confusion with `distinct_count()`.

Reply:
How about
```
number of consecutive groups of equivalent rows in the column
```

Reply:
Thanks! Fixed.